apps/api/src/server/repositories/application.repository.js

```diff
@@ -18,3 +18,63 @@ export const getByStatus = (status) => {
 		}
 	});
 };
+
+/**
+ * @param {string} query
+ * @param {number} skipValue
+ * @param {number} pageSize
+ * @returns {Promise<import('@pins/api').Schema.Application[]>}
+ */
+export const getBySearchCriteria = (query, skipValue, pageSize) => {
+	return databaseConnector.application.findMany({
+		skip: skipValue,
+		take: pageSize,
+		orderBy: [
+			{
+				createdAt: 'desc'
+			}
+		],
+		where: {
+			OR: [
+				{
+					title: { contains: query }
+				},
+				{
+					reference: { contains: query }
+				},
+				{
+					description: { contains: query }
+				}
+			]
+		},
+		include: {
+			subSector: {
+				include: {
+					sector: true
+				}
+			}
+		}
```

Comment:
Do we need to return the sub-sector and sector information?

Reply:
we don't know yet what information we're going to need on the search results page, but if it's any similar to the dashboard we might need to sort the cases by sector

Reply:
As we do not have a design saying the opposite, we are leaving it